posts/mathlib-changelog.md

```diff
@@ -0,0 +1,50 @@
+---
+author: 'David Chanin'
+category: 'Community projects'
+date: 2022-07-27 09:35:23 UTC+02:00
+description: ''
+has_math: false
+link: ''
+slug: mathlib-changelog
+tags: ''
+title: Introducing Mathlib Changelog
+type: text
+---
+
+Tldr; check out [mathlib-changelog.org](https://mathlib-changelog.org) to explore the historical changes to Mathlib, and find out what happened to that lemma you were using.
+
+
+## A Changelog for Mathlib
```

Comment:
```suggestion
## A Changelog for mathlib
```
and throughout. For whatever historical reason, we usually don't capitalize mathlib, and there's precedent in earlier blog posts.